### DISCUSSION TITLE
October 1st Hacktoberfest Stream!

### DISCUSSION THREAD

**Discussion by naomi-lgbt** [EddieHubCommunity/support#2627] [Conference Notes]
This is a discussion where you can introduce yourselves, see who else is attending the event, connect with each other, and share thoughts you learned from our stream!

**Comment by kaiwalyakoparkar** [comment]
Hey everyone :wave:

**Reply by elenajp** [reply]
Hello!!!

**Reply by eddiejaoude** [reply]
yay 🥳

**Reply by JayantBabbar** [reply]
Hey!!!!

**Reply by siddhant-khisty** [reply]
hey

**Reply by atharvashirdhankar** [reply]
Hello nice to meet you 👋

**Reply by tabhay563** [reply]
hey everyone

**Reply by schmelto** [reply]
bojaa @kaiwalyakoparkar  🚀

**Comment by kunal-kushwaha** [comment]
What's good

**Reply by Soham041201** [reply]
Hey kunal

**Reply by siddhant-khisty** [reply]
A PS5 is pretty good.

**Reply by anshumanxda** [reply]
me

**Reply by JayantBabbar** [reply]
Everything is good 🌟

**Reply by aniruddh-bhandarkar** [reply]
Hey

**Reply by dwivedyaakash** [reply]
Hey kunal!

**Reply by PratikDey** [reply]
Hey Kunal

**Comment by siddhant-khisty** [comment]
Hey folks

**Comment by aniket7515** [comment]
Hello everyone

**Comment by Dhananjaymahajan2001** [comment]
hlo all new to opensource looking forward to collaborate and contribute with the community

**Comment by aniruddh-bhandarkar** [comment]
Hello Everyone !

**Comment by aytidao** [comment]
Hello, everyone!

**Comment by hailhydra1** [comment]
hey everyone!

**Comment by iason9** [comment]
Hi Everyone, Happy Hacking and contributing To open source! remember "Contribution 1st Code 2nd"

**Comment by nilabjasarkar123** [comment]
hello everyone

**Comment by shreyaansjain06** [comment]
Hello everyone

**Comment by Soham041201** [comment]
This is my first time in the hactoberfest! Wish me Luck!!

**Comment by Pranai2518** [comment]
Hey everyone

**Comment by guptasajal411** [comment]
Hey folks 👋

**Comment by moksh-pathak** [comment]
Hello everyone!! I'm Moksh, a beginner in Open Source

**Comment by daxoliveira** [comment]
Hi everyone, happy to participate on my first Hacktoberfest!

**Comment by Fahad-Faisaal** [comment]
hello everyone. Just starting out and found this amazing community

**Comment by saisumanthkumar** [comment]
Hey Guys,I regularly face this issue ,
When I'm ready to solve issues ,I'm like I still need to learn something more
Again after learning the gap again I feel the same
Please help me with this

**Reply by kunal-kushwaha** [reply]
Thats okay, learn and contribute.

**Comment by Mitali-13** [comment]
Hi everyone! i am a beginner ! loving the live stream 
some points that i have learned-
1. anyone (even a beginner and non coder) can contribute to open source
2. hacktoberfest is not about getting a tee but getting started with open source
3. join communities to learn and connect
4. check hacktoberfest labels on github, gsoc website for projects to start with

**Comment by atharvashirdhankar** [comment]
Hello everyone👋

**Comment by kkhitesh** [comment]
Hello everyone 👋🏻

**Comment by JayantBabbar** [comment]
Hey everyone!!!

**Comment by rohan-kulkarni-25** [comment]
Hey !  

- shout-out to Eddie and Kunal for letting me know about open source and inspiring me all the time !! 
- Recently Eddie Promoted me as repository maintainer and I am having fun with community 
- Eddie-hub is so welcoming we all greet everyone of us regularly and that keeps us connected all the time 
- And first result of all these thing is that I am having a event as a speaker first time about hackathon and Open Source and I am gonna Say Collaboration First Code Second proudly 
- Learned so  many things from Nick and cahl about reviewing PR and coincidentally my contribution graph is improving 

![image](https://user-images.githubusercontent.com/76584243/135657900-f5142d16-630e-4b61-acbd-82674bbf9e47.png)

**Comment by vikasganiga05** [comment]
Hello everyone :wave:

**Comment by stemount** [comment]
Check out the LIVE YT Video RN @ https://www.youtube.com/watch?v=amz-rJ28IOo

Hello, I am @stemount and I work with companies on their vision, strategy and technical leadership. AMA anything about devops, Kubernetes, Docker, WSL2, I'll probably be able to help, but **create an issue!!** in the support repo so everyone can weigh in.

open source bonus: @Cahllagerfeld review!

![CleanShot 2021-10-01 at 17 46 39](https://user-images.githubusercontent.com/150512/135658687-d85c73c9-da00-40ff-9f44-e456855c5043.png)

![CleanShot 2021-10-01 at 17 46 54 2](https://user-images.githubusercontent.com/150512/135658717-a8d46ab3-c247-45a8-b27c-6e1a8ea6cb2a.png)

**Comment by DGKSK8LIFE** [comment]
Hi everyone 👋

**Reply by stemount** [reply]
DGK in the chat! Let's go, introduce yourself

**Reply by DGKSK8LIFE** [reply]
Hi, I'm Tarek Ali - a student and open-source developer. I'm currently working on [redisql](https://github.com/DGKSK8LIFE/redisql) and some other projects I can't talk about now 🤫. @ me at any of my credentials and I'll respond pretty quickly 🚀!

**Reply by DGKSK8LIFE** [reply]
PS: @stemount is my mentor #1.

**Comment by sanjay-kv** [comment]
Let me say I have never attended any full livestream apart apples live event. The host and Guest speakers were awesome and provided good insights about open source. :)

**Comment by prbhtkumr** [comment]
I do want to get swags but I also want to make valuable contributions to the community.
Still, kinda tough for me to find `good first issues` and projects to contribute to.

**Comment by 0xdeepak** [comment]
Hii everyone!👋

**Comment by prbhtkumr** [comment]
Please post here if you want contributions to your projects. Or want to recommend some. Make sure it has the `hacktoberfest` tags, to be relevant with the discussion.

**Comment by prgayake** [comment]
Hey Evereyone

**Comment by opensourceally** [comment]
Hi Everyone.

**Comment by vd-37** [comment]
Hey everone!

**Comment by vovw** [comment]
helllo

**Comment by Utkarsh-code** [comment]
Heyy, Everyone
It's a really awesome stream;
lots of learning...

**Comment by sidhant-nagpal** [comment]
Hi there everyone! 
Great stream Eddie. It was really insightful.

**Comment by MekongDelta-mind** [comment]
> 
> 
> This is a discussion where you can introduce yourselves, see who else is attending the event, connect with each other, and share thoughts you learned from our stream!

Hey thanks Eddie, Kunal, Nick and Cahl for a great introductory video about the Hacktoberfest and Open source in general.THis will surely help me to start contributing to the Open Source and start my open source journey.

And Hello everyone !! 

I have been working on the Data Analytics and Data Science projects. Please do checkout my portfolio [Jovian Link](https://jovian.ai/pknayak0707-job/overview) and do comment on the same !! 

Happy Contributing !!!

**Reply by MekongDelta-mind** [reply]
> 
> 
> > This is a discussion where you can introduce yourselves, see who else is attending the event, connect with each other, and share thoughts you learned from our stream!
> 
> Hey thanks Eddie, Kunal, Nick and Cahl for a great introductory video about the Hacktoberfest and Open source in general.THis will surely help me to start contributing to the Open Source and start my open source journey.
> 
> And Hello everyone !!
> 
> I have been working on the Data Analytics and Data Science projects. Please do checkout my portfolio [Jovian Link](https://jovian.ai/pknayak0707-job/overview) and do comment on the same !!
> 
> Happy Contributing !!!

The most important learnings from the stream are:
* collaborating is more important then coding
* try to add value rather than counting numbers of lines of code changed
* try to give a detailed description about the issue you are solving so that it is useful for the maintainer and the whole community
